examples/more/external-project-add/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(external-project-add LANGUAGES CXX)
+
+# get external libraries
+SET(GLOBAL_OUTPUT_PATH ${CMAKE_BINARY_DIR}/3rdparty/bin/${CMAKE_BUILD_TYPE})
+include(${PROJECT_SOURCE_DIR}/cmake/Restbed_External.cmake)
```

Comment:
I think for this example, ideally keeping it as simple as possible we could just limit it to one library to add via `ExternalProject_Add`. As this is meant as a learning resource and not a real-world example having this duplication likely isn't adding much. Maybe we only have `Rapidjson` and omit the other two?

Reply:
After going through you comments, I agree; for an example one lib should be enough.